operations/services.md

```diff
@@ -184,6 +184,106 @@ A positive value for `EndOffset` is normal for a nightly stay and implies that t
+
+Returns selected availability and occupancy metrics of a bookable service for a specified time interval, similar to [the availability & occupancy report](https://help.mews.com/s/article/Availability-Occupancy-report). Availability will be returned for all service [time units](services.md#time-unit) that the specified time interval intersects. So, for example, an interval `1st Jan 23:00 UTC - 1st Jan 23:00 UTC` will result in one time unit for `2nd Jan`, while Interval `1st Jan 23:00 UTC - 2nd Jan 23:00 UTC` will result in two time units for `2nd Jan` and `3rd Jan` (assuming a time unit period of "Day"). UTC timestamps must correspond to the start boundary of a [time unit](services.md#time-unit), e.g. 00:00 converted to UTC for a time unit of "Day". Other timestamps are not permitted. The __maximum size of time interval__ depends on the service's [time unit](#time-unit): 367 hours if hours, 367 days if days, or 24 months if months.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/services/getAvailability/2024-01-22`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+    "FirstTimeUnitStartUtc": "2024-02-01T23:00:00.000Z",
+	"LastTimeUnitStartUtc": "2024-02-05T23:00:00.000Z"
+    "Metrics": [
+        "OutOfOrderBlocks",
+        "PublicAvailabilityAdjustment",
+        "OtherServiceReservationCount",
+        "Occupied",
+        "ConfirmedReservations",
+        "OptionalReservations",
+        "BlockAvailability",
+        "AllocatedBlockAvailability",
+        "UsableResources",
+        "ActiveResources"
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) whose availability should be returned. |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](./services.md#time-unit): 367 hours if hours, 367 days if days, or 24 months if months. |
+| `Metrics` | array of string | required | Set of [Service availability metrics](#service-availability-metrics) to be returned. |
```

Comment:
I think we write type like this in case of array

```suggestion
| `Metrics` | array of [Service availability metrics](#service-availability-metrics) | required | Set of [Service availability metrics](#service-availability-metrics) to be returned. |
```